### PR TITLE
Md5 checks

### PR DESCRIPTION
**Pull request by ChefAndy**
- Test Data: updated checksums for modified casemets & altos
- Test: improved test_update_dup_checking to test update logic
- XML models md5 function: New volumes would have the XML string at the
beginning, old files wouldn't
- Ingest: typo in determining if it was a total sync or partial

### CODE REVIEW

File: capstone/capdb/tests/test_ingest.py [hunk 1]
```diff
@@ -26,9 +26,54 @@ def test_case_and_page_xml(volume_xml):
     assert case_xml.pages.count() == 6
 
 @pytest.mark.django_db
-def test_update_dup_checking(volume_xml):
+def test_update_dup_checking(volume_xml, case_xml):
+    fabfile.total_sync_with_s3()
```
Comment (jcushman) [inline capstone/capdb/tests/test_ingest.py]: Nit: redundant of loading the volume_xml fixture (or add a note if that's intentional).

File: capstone/capdb/tests/test_ingest.py [hunk 2]
```diff
@@ -26,9 +26,54 @@ def test_case_and_page_xml(volume_xml):
...
+    original_page_md5 = page_xml.md5()    
+    parsed_page=parse_xml(page_xml.orig_xml)
+    parsed_page('alto|String[ID="ST_15.2.1.5"]').attr["CONTENT"] = "Inversion"
+    page_xml.orig_xml = str(parsed_page)
+    page_xml.save()
+
+    # change corresponding value in casemets
+    parsed_case = parse_xml(case_xml.orig_xml)
+    original_case_md5 = case_xml.md5()
+    parsed_case = parsed_case('casebody|parties[id="b15-4"]').text('The Home Inversion Company of New York v. John Kirk, for use of William Kirk.')
```
Comment (jcushman) [inline capstone/capdb/tests/test_ingest.py]: Nit: `parsed_case = ` shouldn't be in this line -- it means we end up writing just to the `<parties>` element back to orig_xml, which could be confusing later ...

File: capstone/capdb/tests/test_ingest.py [hunk 3]
```diff
@@ -26,9 +26,54 @@ def test_case_and_page_xml(volume_xml):
...
+    assert original_page_md5 != page_xml.md5()
+    assert original_case_md5 != case_xml.md5()
+    assert 'Inversion' in case_xml.orig_xml
+    assert 'Inversion' in page_xml.orig_xml
+
     fabfile.total_sync_with_s3()
-    test_case_and_page_xml(volume_xml)
+
+    # requery items
+    volume_xml = VolumeXML.objects.get(barcode='32044057892259')
```
Comment (jcushman) [inline capstone/capdb/tests/test_ingest.py]: Nit: these can just be `volume_xml.refresh_from_db()`, which makes the intent clearer.

File: capstone/capdb/models.py [hunk 4]
```diff
@@ -24,7 +24,10 @@ def db_type(self, connection):
 
 def checksum(xml):
     m = hashlib.md5()
-    m.update(xml.encode())
+    hashstring = xml
```
Comment (jcushman) [inline capstone/capdb/models.py]: This should be unneeded with my related PR to correctly serialize xml -- can also drop changes to data/1.csv.gz and data/2.csv.gz

**Review by jcushman — COMMENTED** [review]
Nice! Added a couple of nitpicks, and noted a couple of things that can be dropped from this if you land my fix to xml serialization.

**Comment by ChefAndy** [comment]
Need to integrate last PR

**Pull request merged**